static/js/script.js

```diff
@@ -1,3 +1,67 @@
+function toastMessage(tag, message) {
+            // Sets the toast HTML.
```

Comment:
Please make sure that your indentation here is correct.

Reply:
Done